clients/python/sust/api/generated/climate_explorer/__init__.py

```diff
@@ -3,7 +3,7 @@
 """
     Sust Global Climate Explorer API
 
-     This API provides programmatic access to physical risk exposure data. For more guidance on using this API, please visit the Sust Global Dev Center: https://developers.sustglobal.com.   # noqa: E501
+    This API provides programmatic access to physical risk exposure data.For more guidance on using this API, please visit the Sust Global Dev Center: https://developers.sustglobal.com.  # noqa: E501
```

Comment:
Lost one important space in there : `data.For`

Reply:
Did the openapi generator do this?

Reply:
Yeah, the generator did this. I didn't touch any of those files.